leo_segmentation/run.py

```diff
@@ -1,13 +1,38 @@
 # Entry point for the project
+from utils import load_config
+from data import Datagenerator
+from model import Leo
+
+import argparse
 import torch
 
-def train_model(saved_model_path:str):
-    print(f"I am training a model saved at {saved_model_path}")
-    return
+parser = argparse.ArgumentParser(description='Specify train or inference dataset')
+parser.add_argument("-d", "--dataset", type=str, nargs=1, default="pascal_voc")
+args = parser.parse_args()
+dataset = args.dataset[0]
```

Comment:
dataset = args.dataset #dataset[0] is returning first letter

Reply:
![image](https://user-images.githubusercontent.com/13457390/86850679-ff1c2380-c0b1-11ea-88a8-e0916c147e5d.png)
In the command line or IDE it should be ran like this

![image](https://user-images.githubusercontent.com/13457390/86850731-1529e400-c0b2-11ea-9a9a-6bd5c7dc0660.png)

Basically only the dataset name. check if you are getting the same error